src/client_gameloop.rs

```diff
@@ -82,7 +81,7 @@ pub fn gameloop() -> Result<(), String> {
         std::thread::sleep(Duration::new(0, 1_000_000_000u32 / FRAMERATE));
     }
 
-    gs.handle.join().unwrap().unwrap();
+    (move || {gs.handle.join().unwrap().unwrap();})();
```

Comment:
This closure is probably unnecessary at this point

Reply:
yes it could be removed gOOD point